src/ui.rs

```diff
@@ -1370,6 +1381,22 @@ where
         }
     }
 
+    for (out_pin, in_pin, pos) in wire_widgets {
+        let rect = Rect::from_center_size(
+            pos.add(Vec2::from([0.0, -10.0])), // add the widget little above the wire
```

Comment:
Probably can use the same code that is used to calculate rect for popups in egui.